Sources/docc/DocCDocumentation.docc/adding-tables-of-data.md

```diff
@@ -0,0 +1,160 @@
+# Adding Tables of Data
+
+Arrange information into rows and columns.
+
+## Overview
+
+To add a table to your documentation, start a new paragraph and add the following elements, each on its own line:
+
+* A table header, with the heading text of each column, separated by pipes (`|`)
+* A separator row, with at least 3 hyphens (`-`) for each column, separated by pipes
+* One or more rows of table cells, separating each cell of formatted content by a pipe
+ 
+```md
+Sloth speed  | Description                          
+------------ | ------------------------------------- 
+`slow`       | Moves slightly faster than a snail  
+`medium`     | Moves at an average speed           
+`fast`       | Moves faster than a hare            
+`supersonic` | Moves faster than the speed of sound
+```
+
+The example markup above defines the table that's shown below. Each column is automatically sized to fit its widest cell and the table is only as wide as the sum of its columns.
+
+Sloth speed  | Description                          
+------------ | ------------------------------------- 
+`slow`       | Moves slightly faster than a snail  
+`medium`     | Moves at an average speed           
+`fast`       | Moves faster than a hare            
+`supersonic` | Moves faster than the speed of sound
+
+It's not necessary to pad the cells to align the column separators (`|`). However, it may make your table _markup_ easier to read, especially for large or complex tables. 
+
+The same table could have been defined like below. All other examples will uses padded cells for readability.
+
+```md
+Sloth speed|Description
+---|---
+`slow`|Moves slightly faster than a snail
+`medium`|Moves at an average speed
+`fast`|Moves faster than a hare
+`supersonic`|Moves faster than the speed of sound
+```
+
+You can add leading and/or trailing pipes (`|`) if you find that table markup easier to read. This doesn't affect the rendered table on the page. The leading and trailing pipes _can_ be applied inconsistently for each row but doing so may make it harder to discern the structure of the table. 
+
+```md
+| Sloth speed  | Description                          |                         
+| ------------ | ------------------------------------ |
+| `slow`       | Moves slightly faster than a snail   | 
+| `medium`     | Moves at an average speed            |  
+| `fast`       | Moves faster than a hare             |
+| `supersonic` | Moves faster than the speed of sound |
+```
+
+The content of a table cell supports the same style attributes as other text and supports links to other content, including symbols. For more information about styling text, see [Format Text in Bold, Italics, and Code Voice](doc:formatting-your-documentation-content#Format-Text-in-Bold,-Italics,-and-Code-Voice).
```

Comment:
What if I want to include a literal pipe `|` character in my cell content with out impacting the structure of the table? Can I escape pipes using a backslash? `\|` Assuming the answer is yes, it might be worth mentioning that here.

Reply:
I added a sentence that explains how to escape pipe character in table cell content in https://github.com/apple/swift-docc/pull/813/commits/9a377728c9eea1de2f153a9a9bfa72ea97938afa